fastly/resource_fastly_service_v1.go

```diff
@@ -680,6 +680,16 @@ func resourceServiceV1() *schema.Resource {
 							Description:  "Where in the generated VCL the logging call should be placed.",
 							ValidateFunc: validateLoggingPlacement(),
 						},
+						"server_side_encryption": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Description:  "Specify where type of server side encryption should be used.",
```

Comment:
Fix typo `s/where/what/` and please detail what the value can be, i.e. either `AES256` or `aws:kms`.

```suggestion
							Description:  "Specify what type of server side encryption should be used. Can be either `AES256` or `aws:kms`.",
```